play/src/front/Components/Video/VideoMediaBox.svelte

```diff
@@ -46,6 +50,27 @@
 
     onMount(() => {
         resizeObserver.observe(videoContainer);
+        subscribeChangeOutput = speakerSelectedStore.subscribe((deviceId) => {
+            // Check HTMLMediaElement.setSinkId() compatibility for browser
+            // @ts-ignore
+            if (videoElement.setSinkId != undefined) {
+                try {
+                    // @ts-ignore
+                    videoElement.setSinkId(deviceId);
+                } catch (err) {
+                    console.info(
+                        "Your browser is not compatible for updating your speaker over a video element. Try to change the default audio output in your computer settings. Error: ",
+                        err
+                    );
+                }
+            }
+        });
+    });
```

Comment:
I understand the video element will react to a speaker change.

But I don't see the code that is setting the "sinkId" when the video starts. Is this working correctly, and if yes, why?

Reply:
I actually tested it. It is working. But why? :)